cognite_toolkit/cdf_tk/templates/data_classes/_migration_yaml.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+from collections import UserList, defaultdict
+from collections.abc import Collection
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, cast
+
+from rich import print
+from rich.markdown import Markdown
+from rich.panel import Panel
+
+from cognite_toolkit._version import __version__ as current_version
+from cognite_toolkit.cdf_tk.templates import COGNITE_MODULES
+from cognite_toolkit.cdf_tk.templates._utils import iterate_modules
+from cognite_toolkit.cdf_tk.utils import load_yaml_inject_variables
+
+
+@dataclass
+class Change:
+    title: str
+    steps: list[str]
+
+    @classmethod
+    def load(cls, data: dict[str, Any]) -> Change:
+        return cls(
+            title=data["title"],
+            steps=data["steps"],
+        )
+
+    def print(self) -> None:
+        print(Panel.fit(self.title, title="Change"))
```

Comment:
It would be useful to separate each change also with a newline above the header.